.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  test:
```

Comment:
Please put the coveralls stuff back into this file - I've added the secret to the repo so that should work now.